test/helpers.spec.ts

```diff
@@ -159,6 +190,94 @@ describe('helpers', () => {
         });
       });
 
+      describe('enumValue', () => {
+        enum ColorValueEnum {
+          Red,
+          Green,
+          Blue,
+        }
+        enum ColorValueWithStartIndexEnum {
+          Red = 3,
+          Green,
+          Blue,
+        }
+        enum ColorStringEnum {
+          Red = 'RED',
+          Green = 'GREEN',
+          Blue = 'BLUE',
+        }
+        enum FooMixedEnum {
+          Foo = 0,
+          Bar = 1,
+          StrFoo = 'FOO',
+          StrBar = 'BAR',
+        }
+
+        it('should return a value from a numeric enum', () => {
+          const randomColorValue = faker.helpers.enumValue(ColorValueEnum);
+          expect([
+            ColorValueEnum.Red,
+            ColorValueEnum.Green,
+            ColorValueEnum.Blue,
+          ]).toContain(randomColorValue);
+        });
+
+        it('should return a value from a numeric enum that first value is not 0', () => {
+          const randomColorVAlue = faker.helpers.enumValue(
+            ColorValueWithStartIndexEnum
+          );
+          expect([
+            ColorValueWithStartIndexEnum.Red,
+            ColorValueWithStartIndexEnum.Green,
+            ColorValueWithStartIndexEnum.Blue,
+          ]).toContain(randomColorVAlue);
+        });
+
+        it('should return a value from a string enum', () => {
+          const randomColorVAlue = faker.helpers.enumValue(ColorStringEnum);
```

Comment:
Please fix the typo here and in the other methods:
```suggestion
          const randomColorValue = faker.helpers.enumValue(ColorStringEnum);
```

Alternatively we sometimes use `actual` as variable name.